src/content/en/updates/posts/2015/07/interact-with-ble-devices-on-the-web.markdown

```diff
@@ -58,17 +58,22 @@ Bluetooth characteristics.
 
 ### HTTPS Only
 
-Because the Web Bluetooth API is a powerful new feature added to the Web,
+Because this experimental API is a powerful new feature added to the Web,
 Google Chrome aims to make it available only to [secure
-contexts](https://w3c.github.io/webappsec/specs/powerfulfeatures/#intro).  This
+contexts](https://w3c.github.io/webappsec/specs/powerfulfeatures/#intro). This
 means you'll need to build with TLS in mind.
 
-During development you'll be able to play with Web Bluetooth through
-http://localhost by using some tools like the [Chrome Dev
-Editor](https://chrome.google.com/webstore/detail/chrome-dev-editor-develop/pnoffddplpippgcfjdhbmhkofpnaalpg),
-but to deploy it on a site you'll need to have HTTPS setup
-on your server. I personally enjoy [GitHub Pages](https://pages.github.com) for
-demo purposes.
+> We care deeply about security, so you will notice that all new Web APIs
```

Comment:
Thanks! I'll use "new web capabilities" instead of "all new Web APIs".
I use this quote because I wanted to emphasize.